### PR TITLE
Natasha/recast

### PR DESCRIPTION
**Pull request by natashaarmbrust**
Changing recommend to recast, updating app name!

### CODE REVIEW

**Comment by natashaarmbrust** [comment]
img_1098
img_1099
img_1100
img_1101

File: Podcast/EpisodeSubjectView.swift [hunk 1]
```diff
@@ -121,7 +121,8 @@ class EpisodeSubjectView: UIView {
         }
 
         descriptionLabel.snp.makeConstraints { make in
-            make.top.equalTo(podcastImage.snp.bottom).offset(descriptionLabelOffset)
+            make.top.greaterThanOrEqualTo(podcastImage.snp.bottom).offset(descriptionLabelOffset)
+            make.top.greaterThanOrEqualTo(dateTimeLabel.snp.bottom).offset(descriptionLabelOffset)
```
Comment (mindylou) [inline Podcast/EpisodeSubjectView.swift:125]: nice!!

Comment (mindylou) [reply Podcast/EpisodeSubjectView.swift:125]: i'm still getting constraint errors on this line though with the `EpisodeUtilityBarButton` height

**Review by mindylou — COMMENTED** [review]
LGTM! just one thing about the constraint error in `EpisodeSubjectView`

**Pull request merged**